doc/reference/drivers/index.rst

```diff
@@ -104,14 +104,14 @@ split into read-only and runtime-mutable parts. At a high level we have:
 
   struct device {
 	const char *name;
-	const void *config_info;
+	const void *config;
```

Comment:
This patch is out of date: there are new `config_info` mentions relative to `<sys/device_mmio.h>` that need to be changed.

Reply:
yes please rebase and kick off CI again, should catch anything missed since x86 uses this stuff